package.json

```diff
@@ -34,10 +41,10 @@
   "devDependencies": {
     "@types/deep-extend": "0.4.32",
     "@types/jest": "^27.0.2",
-    "@types/node": "^16.11.6",
+    "@types/node": "^16.18.0",
     "@typescript-eslint/parser": "^5.3.0",
     "@vercel/ncc": "^0.31.1",
-    "eslint": "^7.32.0",
+    "eslint": "^8.39.0",
```

Comment:
Yarn 3 gives more information and it came up with this new warning:

```js
$ yarn install

➤ YN0000: ┌ Resolution step
➤ YN0060: │ datadog-synthetics-github-action@workspace:. provides eslint (pe1b28) with version 7.32.0, which doesn't satisfy what eslint-plugin-github and some of its descendants request
➤ YN0000: │ Some peer dependencies are incorrectly met; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code
➤ YN0000: └ Completed
➤ YN0000: ┌ Fetch step
➤ YN0000: └ Completed
➤ YN0000: ┌ Link step
➤ YN0000: └ Completed
➤ YN0000: Done with warnings in 0s 491ms
```

```js
$ yarn explain peer-requirements pe1b28

➤ YN0000: datadog-synthetics-github-action@workspace:. provides eslint@npm:7.32.0 with version 7.32.0, which doesn't satisfy the following requirements:

➤ YN0000: @typescript-eslint/eslint-plugin@npm:5.3.0 [09fd5]      → ^6.0.0 || ^7.0.0 || ^8.0.0                 ✓
➤ YN0000: @typescript-eslint/experimental-utils@npm:5.3.0 [6251a] → *                                          ✓
➤ YN0000: @typescript-eslint/parser@npm:5.3.0 [09fd5]             → ^6.0.0 || ^7.0.0 || ^8.0.0                 ✓
➤ YN0000: eslint-config-prettier@npm:8.3.0 [09fd5]                → >=7.0.0                                    ✓
➤ YN0000: eslint-plugin-eslint-comments@npm:3.2.0 [09fd5]         → >=4.19.1                                   ✓
➤ YN0000: eslint-plugin-filenames@npm:1.3.2 [09fd5]               → *                                          ✓
➤ YN0000: eslint-plugin-github@npm:4.3.3 [7fc47]                  → ^8.0.1                                     ✘
➤ YN0000: eslint-plugin-i18n-text@npm:1.0.1 [09fd5]               → >=5.0.0                                    ✓
➤ YN0000: eslint-plugin-import@npm:2.25.2 [09fd5]                 → ^2 || ^3 || ^4 || ^5 || ^6 || ^7.2.0 || ^8 ✓
➤ YN0000: eslint-plugin-prettier@npm:3.4.1 [09fd5]                → >=5.0.0                                    ✓
➤ YN0000: eslint-utils@npm:3.0.0 [e1cc4]                          → >=5                                        ✓

➤ YN0000: Note: these requirements start with eslint-plugin-github@npm:4.3.3 [7fc47]
```

Upgrading to eslint@8 fixes this issue.